.github/actions/terraform_state_artifact/action.yml

```diff
@@ -0,0 +1,202 @@
+
+
+        if ($tf_workflow_sid_assign_to_anyone) {
+          terraform import module.routing.twilio_taskrouter_workspaces_workflows_v1.assign_to_anyone $tf_workspace_sid/$tf_workflow_sid_assign_to_anyone
+        }
+
+        if ($tf_workflow_sid_chat_transfer) {
+          terraform import module.routing.twilio_taskrouter_workspaces_workflows_v1.chat_transfer $tf_workspace_sid/$tf_workflow_sid_chat_transfer
+        }
+
+        if ($tf_workflow_sid_callback) {
+          terraform import module.routing.twilio_taskrouter_workspaces_workflows_v1.callback $tf_workspace_sid/$tf_workflow_sid_callback
+        }
+
+        if ($tf_workflow_sid_internal_call) {
+          terraform import module.routing.twilio_taskrouter_workspaces_workflows_v1.internal_call $tf_workspace_sid/$tf_workflow_sid_internal_call
+        }
+
+        # queues
+        twilio api:taskrouter:v1:workspaces:task-queues:list --workspace-sid $tf_workspace_sid | select-string "Everyone" | ForEach-Object { ([string]$_).Split(" ")[0] } | tee-object -variable tf_queue_sid_everyone
+        twilio api:taskrouter:v1:workspaces:task-queues:list --workspace-sid $tf_workspace_sid | select-string "Sales" | ForEach-Object { ([string]$_).Split(" ")[0] } | tee-object -variable tf_queue_sid_sales
+        twilio api:taskrouter:v1:workspaces:task-queues:list --workspace-sid $tf_workspace_sid | select-string "Support" | ForEach-Object { ([string]$_).Split(" ")[0] } | tee-object -variable tf_queue_sid_support
+        twilio api:taskrouter:v1:workspaces:task-queues:list --workspace-sid $tf_workspace_sid | select-string "Internal Calls" | ForEach-Object { ([string]$_).Split(" ")[0] } | tee-object -variable tf_queue_sid_internal_calls
+
+
+        if ($tf_queue_sid_everyone) {
+          terraform import module.routing.twilio_taskrouter_workspaces_task_queues_v1.everyone $tf_workspace_sid/$tf_queue_sid_everyone
+        }
+
+        if ($tf_queue_sid_sales) {
+          terraform import module.routing.twilio_taskrouter_workspaces_task_queues_v1.sales $tf_workspace_sid/$tf_queue_sid_sales
+        }
+
+        if ($tf_queue_sid_support) {
+          terraform import module.routing.twilio_taskrouter_workspaces_task_queues_v1.support $tf_workspace_sid/$tf_queue_sid_support
+        }
+
+        if ($tf_queue_sid_internal_calls) {
+          terraform import module.routing.twilio_taskrouter_workspaces_task_queues_v1.internal_calls $tf_workspace_sid/$tf_queue_sid_internal_calls
+        }   
+
+        # channels
+        twilio api:taskrouter:v1:workspaces:task-channels:list --workspace-sid $tf_workspace_sid | select-string "Voice" | ForEach-Object { ([string]$_).Split(" ")[0] } | tee-object -variable tf_task_channel_sid_voice
+        twilio api:taskrouter:v1:workspaces:task-channels:list --workspace-sid $tf_workspace_sid | select-string "Chat" | ForEach-Object { ([string]$_).Split(" ")[0] } | tee-object -variable tf_task_channel_sid_chat
```

Comment:
These channel names need to be a full match; in my environment both picked incorrect channels.